docs/docs/dev-guide-new-shield.md

```diff
@@ -393,7 +393,6 @@ The two `#include` lines at the top of the keymap are required in order to bring
 Further documentation on behaviors and bindings is forthcoming, but a summary of the current behaviors you can bind to key positions is as follows:
 
 - `kp` is the "key press" behavior, and takes a single binding argument of the HID keycode from the 'keyboard/keypad" HID usage table.
```

Comment:
> and takes a single binding argument of the HID keycode from the 'keyboard/keypad" HID usage table.

Does this need generalizing?

Reply:
I changed the wording a bit to be more accurate, but left the term "keycode" in parentheses to help people understand.

Reply:
Given that this particular document is aimed at people who are unlikely to care about what the code is made of (HID parts), I'd just keep it simple.  Perhaps just `... for key codes` ... that is until we come up with a better work to replace it (something more encompassing that includes other controls).

Reply:
Updated.